packages/web-components/src/components/masthead/masthead-composite.ts

```diff
@@ -68,6 +72,85 @@ enum NAV_ITEMS_RENDER_TARGET {
  */
 @customElement(`${ddsPrefix}-masthead-composite`)
 class DDSMastheadComposite extends LitElement {
+  private _renderL1Items({ target }: { target: NAV_ITEMS_RENDER_TARGET }) {
+    if (!this.l1Data) return undefined;
+    const { menuItems } = this.l1Data;
+    if (menuItems) {
+      return target === NAV_ITEMS_RENDER_TARGET.TOP_NAV
+        ? html`
+            <dds-top-nav hide-divider>
+              ${menuItems.map((elem: MastheadL1Item, i) => {
+                return elem.menuItems
+                  ? html`
+                      <dds-top-nav-menu
+                        menu-label="${elem.title}"
+                        trigger-content="${elem.title}"
+                        data-autoid="${ddsPrefix}--masthead__l1-nav--nav-${i}"
+                      >
+                        ${elem.menuItems.map(
+                          (item, j) => html`
+                            <dds-top-nav-menu-item
+                              href="${item.url}"
+                              title="${item.title}"
+                              data-autoid="${ddsPrefix}--masthead__l1-nav--subnav-col${i}-item${j}"
+                            ></dds-top-nav-menu-item>
+                          `
+                        )}
+                      </dds-top-nav-menu>
+                    `
+                  : html`
+                      <dds-top-nav-item
+                        href="${elem.url}"
+                        title="${elem.title}"
+                        data-autoid="${ddsPrefix}--masthead__l1-nav--nav-${i}"
+                      ></dds-top-nav-item>
+                    `;
+              })}
+            </dds-top-nav>
+          `
+        : menuItems.map((elem: MastheadL1Item, i) =>
+            elem.menuItems
+              ? html`
+                  <dds-left-nav-menu title="${elem.title}" data-autoid="${ddsPrefix}--masthead__l1-sidenav--nav-${i}">
+                    ${elem.menuItems.map(
+                      (item, j) => html`
+                        <dds-left-nav-menu-item
+                          href="${item.url}"
+                          title="${item.title}"
+                          data-autoid="${ddsPrefix}--masthead__l1-sidenav--subnav-col${i}-item${j}"
+                        ></dds-left-nav-menu-item>
+                      `
+                    )}
+                  </dds-left-nav-menu>
+                `
+              : html`
+                  <dds-left-nav-item
+                    href="${elem.url}"
+                    title="${elem.title}"
+                    data-autoid="${ddsPrefix}--masthead__l1-sidenav--nav-${i}"
+                  ></dds-left-nav-item>
+                `
+          );
+    }
+
+    return undefined;
+  }
+
+  private _renderL1() {
```

Comment:
Could you please add JSDoc comment here?

Reply:
(This is for striving for easier future maintenance with well-documented codebase)